recipe-server/normandy/settings.py

```diff
@@ -351,6 +351,8 @@ def RAVEN_CONFIG(self):
     AWS_SECRET_ACCESS_KEY = values.Value()
     AWS_STORAGE_BUCKET_NAME = values.Value()
 
+    GITHUB_URL = values.Value('https://github.com/mozilla/normandy')
```

Comment:
This setting is also used in our `/__version__` endpoint. I think it's fine to have it be null for a moment. This is an uncommon case, and we should be fetching service info on every page any way, since it powers other parts of the UI.